docs/en/01_Working_with_projects/01_Setting_up_your_project.md

```diff
@@ -60,7 +60,7 @@ You will need to make your first commit to Git and push your project into your G
 
 As mentioned you should not commit the packages of code managed by Composer to your project. To ensure this you need to use a `.gitignore` file stored in the root of your project (you should already have one of these files from the installation process).
 
-Inside the `.gitignore` you store references to the folders in your project you DO NOT want commited to your project Git repository (these should be the same packages of code referenced in your composer.json file).
+Inside the `.gitignore` you store references to the folders in your project you DO NOT want committed to your project Git repository (these should be the same packages of code referenced in your composer.json file).
```

Comment:
I think we can remove the bracketed part here, since most/all CWP 2 modules will be in the vendor folder and be ignored by default